### PR TITLE
[KEYBOARD] Add Swiss French keyboard layout

### PR DESCRIPTION
**Pull request by ReiatsuSan**
## Purpose

The swiss frech keyboard layout was missing

JIRA issue: [CORE-XXXX](https://jira.reactos.org/browse/CORE-XXXX)

## Proposed changes

It adds the required files and references to the hive file

- 
- 

## TODO

None atm

### CODE REVIEW

File: dll/keyboard/kbdsf/kbdsf.c [hunk 1]
```diff
@@ -0,0 +1,479 @@
+/*
+ * ReactOS French (Switzerland) ASCII Keyboard layout
+ * Copyright (C) 2005 ReactOS
+ * License: LGPL, see: LGPL.txt
```
Comment (binarymaster) [inline dll/keyboard/kbdsf/kbdsf.c:4]: As you are copying the header from neighbor files, I noticed the Author field is missing, please fill it in. Also suggesting to specify current year for copyright.
```suggestion
 * Copyright (C) 2020 ReactOS
 * Author: ???
 * License: LGPL, see: LGPL.txt
```

**Review by binarymaster — CHANGES_REQUESTED** [review]
Hello,
Thanks for your interest in ReactOS!
We ask our contributors [to use their full name when committing](https://github.com/reactos/reactos/blob/master/CONTRIBUTING.md#rules-and-recommendations).
[Please amend your commit with your full name](https://stackoverflow.com/a/3042512), and update this PR.

Currently you are committing changes under a nickname `ReiatsuSan`.

To change this for future PR's, you can update it for globally (for the system):
```
git config --global user.name "Your Name"
git config --global user.email you@example.com
```

Or just for the current repository:
```
git config user.name "Your Name"
git config user.email you@example.com
```

Also please update your name in [GitHub profile settings](https://github.com/settings/profile).

Regards,

**Comment by ReiatsuSan** [comment]
Hi, sorry I am new with Git stuff. Let me know if I made something wrong.
I think itshould be correct now

File: dll/keyboard/kbdsf/kbdsf.spec [hunk 2]
```diff
@@ -0,0 +1 @@
+@ stdcall KbdLayerDescriptor()
```
Comment (binarymaster) [inline dll/keyboard/kbdsf/kbdsf.spec:1]: No newline at end of file, please add.

**Review by binarymaster — COMMENTED** [review]
Now it looks good, thanks. Added one more review comment, but it should be reviewed by some more developers I think.

File: dll/keyboard/CMakeLists.txt [hunk 3]
```diff
@@ -60,6 +60,7 @@ list(APPEND _keyboard_layouts
     kbdru
     kbdru1
     kbdsg
+    kbdsf
```
Comment (HBelusca) [inline dll/keyboard/CMakeLists.txt:63]:
```suggestion
    kbdsf
    kbdsg
```
(classify by lexicographical order)

File: boot/bootdata/hivesys.inf [hunk 4]
```diff
@@ -570,6 +570,10 @@ HKLM,"SYSTEM\CurrentControlSet\Control\Keyboard Layouts\0000040c","Layout File",
 HKLM,"SYSTEM\CurrentControlSet\Control\Keyboard Layouts\0000040c","Layout Text",0x00000000,%FRENCH%
 HKLM,"SYSTEM\CurrentControlSet\Control\Keyboard Layouts\0000040c","Layout Display Name",0x00000000,"@%SystemRoot%\system32\input.dll,-5010"
 
+HKLM,"SYSTEM\CurrentControlSet\Control\Keyboard Layouts\0000100c","Layout File",0x00000000,"kbdsf.dll"
+HKLM,"SYSTEM\CurrentControlSet\Control\Keyboard Layouts\0000100c","Layout Text",0x00000000,%SWISS_FRENCH%
+HKLM,"SYSTEM\CurrentControlSet\Control\Keyboard Layouts\0000100c","Layout Display Name",0x00000000,"@%SystemRoot%\system32\input.dll,-5023"
+
```
Comment (HBelusca) [inline boot/bootdata/hivesys.inf:576]: Please put these entries after the canadian_french entries below (lexicographical order)

File: dll/keyboard/kbdsf/kbdsf.c [hunk 5]
```diff
@@ -0,0 +1,479 @@
+/*
```
Comment (HBelusca) [inline dll/keyboard/kbdsf/kbdsf.c:1]: Can you tell me whether this file has been encoded in UTF-8 or not? (If not, please do that.)

Comment (ReiatsuSan) [reply dll/keyboard/kbdsf/kbdsf.c:1]: The file is already UTF8 encoded

Comment (HBelusca) [reply dll/keyboard/kbdsf/kbdsf.c:1]: ok thanks. (I was asking because I saw characters that were strangely displayed here.)

**Review by binarymaster — APPROVED** [review]
Ok, LGTM. Note that I force-pushed to your branch in order to avoid merge commits. Next time please create PRs from non-master branches.

**Review by HBelusca — APPROVED** [review]

**Pull request merged**